refarch-integrations/refarch-s3-integration/pom.xml

```diff
@@ -19,8 +19,12 @@
     <modules>
         <module>refarch-s3-integration-core</module>
         <module>refarch-s3-integration-starter</module>
-        <module>refarch-s3-integration-service</module>
-        <module>refarch-s3-integration-client</module>
-        <module>refarch-s3-integration-client-starter</module>
+        <module>refarch-s3-integration-client-core</module>
+        <!-- REST -->
+        <module>refarch-s3-integration-rest-core</module>
+        <module>refarch-s3-integration-rest-starter</module>
+        <module>refarch-s3-integration-rest-service</module>
+        <module>refarch-s3-integration-rest-client</module>
+        <module>refarch-s3-integration-rest-client-starter</module>
```

Comment:
Might it be more suitable to move all those modules into a submodule `refarch-s3-integration-rest`? Then we have one submodule for each integration type.

Reply:
Done